build.proj

```diff
@@ -261,6 +262,10 @@
         ExpectedDelaySigned="false"
         ContinueOnError="false" 
         Condition="!$(DelaySign) and '@(DelaySignedAssembliesToSign)' != ''"/>
+		
+	<VerifyAuthenticodeSignatureTask SignedFiles="@(DelaySignedAssembliesToSign);@(ScriptsToSign)">
```

Comment:
This is already done in the signing task - what we need is to include the files that we did not build (those files copied by the compiler, usua;lly from nuget packages)